andela/travel_tool_front
# #160106281 Fix text editor bug

#### What does this PR do?
This PR fixes the missing placeholder text in WYSIWYG text editor on the request modal 
#### Description of Task to be completed?
- Implement  placeholder text in the text editor 
- Write the test  for the changed method

#### How should this be manually tested?
- Clone the repository
- Run `make start` docker build command on the terminal to install all dependencies and run the App in the docker development environment server.
- On the browser, type the localhost URL ` http://localhost:3000` and Login with your Andela email account into the App.
- On the Requests page, click on the request Id from the list of requests displayed on the request table, you should be able to see the Request details Modal displayed with the text editor having a placeholder text in it.


#### Any background context you want to provide?
To test if this works, you will need both the [backend(https://github.com/andela/travel_tool_back)] API and frontend client set up locally or you can test it on the staging environment.

#### What are the relevant pivotal tracker stories?
[#160106281](https://www.pivotaltracker.com/story/show/160106281)


#### Screenshots (if appropriate)

<img width="1280" alt="screen shot 2018-08-30 at 21 17 40" src="https://user-images.githubusercontent.com/12033494/44871304-1494bb00-ac9b-11e8-815e-ddf29fa528c1.png">



#### Questions:
None
